core/client/app/controllers/setup/two.js

```diff
@@ -72,14 +73,13 @@ export default Ember.Controller.extend(ValidationEngine, {
                     config.set('blogTitle', data.blogTitle);
                     // Don't call the success handler, otherwise we will be redirected to admin
                     self.get('application').set('skipAuthSuccessHandler', true);
-
                     self.get('session').authenticate('simple-auth-authenticator:oauth2-password-grant', {
                         identification: self.get('email'),
                         password: self.get('password')
                     }).then(function () {
                         self.set('password', '');
                         self.set('blogCreated', true);
-
+                        self.get('controllers.setup').set('blogCreated', true);
```

Comment:
Rather than manually setting `blogCreated` on both `this` and `controllers.setup` you could define the relationship up-front by doing `blogCreated: Ember.computed.alias('controllers.setup.blogCreated')` making the `setup` controller the single source of truth.